src/schema/auth/user.ts

```diff
@@ -34,13 +83,20 @@ export const UserMutation = extendType({
         t.field('addUser', {
             type: User,
             args: {
-                id: nonNull(stringArg()),
+                id: stringArg(),
                 username: nonNull(stringArg()),
                 email: nonNull(stringArg()),
             },
             // args typen i resolver er mappa til args typen definnert over
             resolve: async (_, args, ctx) => {
-                const user = await ctx.prisma.user.create({ data: { ...args } });
+                const user =
+                    args.id !== null && args.id !== undefined
```

Comment:
Can do `!!args.id` to convert it into a false (if null/undefined) and true if it exists. I also think that id should be nonNull arg so we dont need to do this check

Reply:
What you propose sounds reasonable. The idea is that the userId should be the same as the authId (auth.sub from auth0). This means that when a user is created after the user is created in auth0, the id should be provided and set. A user should also be  able to be created without any link to an auth0 user, so that an administrator can add a user to a meeting before they have registered. In this case there will be no auth0-id to use, so the id should be given randomly. 

Better solutions then the existing is welcomed, but we should support both generation id and setting id.